fhir-path/src/main/java/com/ibm/fhir/path/tools/Main.java

```diff
@@ -0,0 +1,372 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.path.tools;
```

Comment:
in other projects, I think I used `*.tool` in the package (singular vs plural).  Not sure how important it is to be consistent on this though

Reply:
Interesting... I looked in another project and it had tools.  might be a good cleanup item